### DISCUSSION TITLE
config.ini

### DISCUSSION THREAD

**Discussion by Daveolw** [raspicamplayer/camplayer#60] [Camplayer on Raspberry Pi OS]
In Which Directory should the config.ini go?
Followed instructions and get this error,
'/home/pi/.camplayer/config.ini' not found!
No configuration file found

**Comment by cybermaus** [comment]
You need to create it, for example by copying the example config, or by creating it with `touch /home/pi/.camplayer/config.ini`

Or just edit it with your favourite editor, maybe `nano` or `vi`, and then upon exit save it. That will also create the file

**Comment by Daveolw** [comment]
That is what I did as per instructions, 

Create and open following file (assumes you are running as user 'pi')

/home/pi/.camplayer/config.ini



I used Raspberry terminal    sudo nano /home/pi/.camplayer/config.ini

and then saved it, but it cannot find

**Reply by OzRocky** [reply]
Just a guess, but try without sudo. You are probably logged in as the Pi user so you already own the home folder for that user.
